Entities/LevelStats.cs

```diff
@@ -75,20 +75,26 @@ public class LevelStats {
             { "round_lava",                       new LevelStats("Slime Climb", LevelType.Race, false, 1) },
             { "round_tip_toe",                    new LevelStats("Tip Toe", LevelType.Race, false, 1) },
             { "round_gauntlet_03",                new LevelStats("Whirlygig", LevelType.Race, false, 1) },
+            { "round_gauntlet_04",                new LevelStats("Knight Fever", LevelType.Race, false, 1) },
+            { "round_wall_guys",                  new LevelStats("Wall Guys", LevelType.Race, false, 1) },
 
             { "round_block_party",                new LevelStats("Block Party", LevelType.Survival, false, 1) },
             { "round_jump_club",                  new LevelStats("Jump Club", LevelType.Survival, false, 1) },
             { "round_match_fall",                 new LevelStats("Perfect Match", LevelType.Survival, false, 1) },
             { "round_tunnel",                     new LevelStats("Roll Out", LevelType.Survival, false, 1) },
             { "round_tail_tag",                   new LevelStats("Tail Tag", LevelType.Survival, false, 1) },
 
+            { "round_hoops_blockade_solo",        new LevelStats("Hoopsie Legends", LevelType.Hunt, false, 1) },
+
```

Comment:
I like the new LevelType ❤️ 
Please set it to Royal Fumble too!

Reply:
Wait, is Royal Fumble a hunt level now?, Tail Tag, Team Tail Tag and Jinxed too?

Reply:
No. It's a final, but we deleted `Final` as a `LevelType` because we need that each final can show a different stat (fastest, longest) as default.

Reply:
So.. i.e. `Fall Mountain` is a `Race` and have the `isFinal = true` flag